packages/eds-core-react/src/components/Button/Button.stories.tsx

```diff
@@ -308,6 +310,12 @@ export const Compact: Story<ButtonProps> = () => {
       <Button variant="ghost_icon">
         <Icon data={menu} title="Ghost icon menu"></Icon>
       </Button>
+      <Button.Group ria-label="button group compact">
```

Comment:
missing a in aria

Reply:
done

Reply:
Ok you misunderstood slightly; what i mean is above here is this (ria label):  <Button.Group ria-label="button group compact">
I don't think you need aria label on the buttons

Reply:
Oh i see a typo! 
Every button with icon should have aria-label. [Here](https://medium.com/100-days-in-kyoto-to-create-a-web-app-with-google/day-16-icon-buttons-should-be-labelled-with-aria-label-1056515d3707)

Reply:
Oh true 😄